controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -204,7 +204,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                                                 false, delegationToken, requestId), executor));
     }
 
-    private CompletableFuture<Boolean> isDeleted(String scope, String kvtName, KVTOperationContext context) {
```

Comment:
why is this being removed?

Reply:
It was passed but not used.

Reply:
it was passed but not used

Reply:
Fie this for the delete test failure: https://github.com/pravega/pravega/issues/4927